lib/routes/mp4er/index.js

```diff
@@ -59,7 +59,14 @@ module.exports = async (ctx) => {
 
                 const torrents = content('#torrent-list .list');
 
-                item.description = content('.info0').html() + `<div><b>下载地址：</b>${downloadLinks}</div>` + `${torrents.html() ? `<div><b>种子列表：</b>${torrents.html()}</div>` : ''}`;
+                item.description = `
+                    ${content('.info0').html()}
+                    <div>
+                        <b>下载地址：</b>
+                        ${downloadLinks}
+                    </div>
+                    ${torrents.html() ? `<div><b>种子列表：</b>${torrents.html()}</div>` : ''}
+                `;
```

Comment:
This might create unnecessary space in the output.

Reply:
Does this effect the rendered content?
I thought that the source code could be more easy to read.
So what is your suggestion?

Reply:
[`094849a` (#8175)](https://github.com/DIYgod/RSSHub/pull/8175/commits/094849a4552f735964e00b0e345fd1027f303f87)

How about this?

Reply:
LGTM then~